devimint/src/cfg/lnd.conf

```diff
@@ -4,6 +4,10 @@ listen=0.0.0.0:{listen_port}
 rpclisten=0.0.0.0:{rpc_port}
 restlisten=0.0.0.0:{rest_port}
 noseedbackup=1
+wtclient.active=false
+sync-freelist=true
```

Comment:
These look good. I was also playing with setting `reset-wallet-transactions=false` for restarts, but not necessary if we see fast startups.

https://github.com/lightningnetwork/lnd/blob/master/sample-lnd.conf#L346